code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -0,0 +1,250 @@
+	playsound(src,'sound/magic/lightningbolt.ogg', 30, TRUE)
+	new /obj/effect/decal/cleanable/ash(get_turf(src))
+	qdel(src)
+
+
+/obj/effect/proc_holder/spell/targeted/force_lightning
+	name = "Force Lightning"
+	desc = "Release a blast of force lightning at a random target within 4 tiles. Arcs between targets, and decays in power with each arc."
+	charge_type = "recharge"
+	charge_max	= 10
+	clothes_req = FALSE
+	range = 4
+	cooldown_min = 10
+	selection_type = "view"
+	random_target = TRUE
+	invocation = null
+	invocation_type = null
+	var/static/mutable_appearance/halo
+	var/sound/Snd // so far only way i can think of to stop a sound, thank MSO for the idea.
+	var/bolt_flags = SHOCK_NOGLOVES | SHOCK_NOSTUN
+	var/max_bounces = 10
+	var/bolt_decay_per_bounce = 4 //How much damage does the bolt lose when it bounces.
+	var/min_damage = 5
+	var/max_damage = 20
+	var/stamina_damage_multiplier = 0.5
+	var/datum/effect_system/spark_spread/spark_system
+	var/list/existing_targets = list()
+
+	action_icon_state = "lightning"
+
+/obj/item/clothing/head/helmet/chaplain/sith
+	name = "sith hood"
+	desc = "It stinks of old man, ozone and the Dark Side."
+	icon_state = "crusader"
+	item_state = "crusader"
+	flags_cover = HEADCOVERSEYES
+	flags_inv = HIDEHAIR|HIDEFACE|HIDEEARS
+
+/obj/item/clothing/suit/armor/riot/chaplain/sith
+	name = "sith robes"
+	desc = "They stink of old man, ozone and the Dark Side."
+	icon_state = "crusader"
+	item_state = "crusader"
+
+
+/obj/effect/proc_holder/spell/targeted/force_lightning/proc/spark_setup()
+	spark_system = new /datum/effect_system/spark_spread()
+	spark_system.set_up(5, 0, src)
+
+
+/obj/effect/proc_holder/spell/targeted/force_lightning/cast(list/targets, mob/user = usr)
+	if(!can_cast(user))
+		return
+
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		if(C.handcuffed)
+			to_chat(user, "<span class='warning'>You can't use [src] while handcuffed!</span>")
+			return
+
+	existing_targets = list()
+	if(!targets.len)
+		to_chat(user, "<span class='warning'>No target found in range!</span>")
+		return
+	spark_setup()
+	user.say(pick("My hate has made me powerful...","I AM THE SENATE!!", "If you will not be turned... you will be destroyed!!", "You will pay the price for your lack of vision!!", "Young fool, only now, at the end, do you understand.", "Your feeble skills are no match for the power of the Dark Side!", "UNNNNLIMITEDDDD POWAHHHH!!", "And now young Skywalker, you will die."))
+	var/mob/living/carbon/target = targets[1]
+	if(get_dist(user,target)>range)
+		to_chat(user, "<span class='warning'>[target.p_theyre(TRUE)] too far away!</span>")
+		return
+
+	playsound(get_turf(user), 'sound/magic/lightningbolt.ogg', 50, TRUE)
+	user.Beam(target,icon_state="lightning[rand(1,12)]",time=5)
+
+	var/damage = rand(min_damage,max_damage)
+
+	if(isliving(user))
+		var/mob/living/L = user
+		L.adjustStaminaLoss(damage * stamina_damage_multiplier) //Take stamina damage equal to half the amount of electrical damage.

```

Comment:
IMO, the stamina damage should be MORE than the normal damage. If the stamina damage is only half of the "real" damage, then your victim(s) will be dead before they ever enter staminacrit.